src/backend/aspen/api/settings.py

```diff
@@ -166,11 +166,11 @@ def EXTERNAL_AUSPICE_BUCKET(self) -> str:
     ####################################################################################
     # sentry properties
     @cached_property
-    def SENTRY_URL(self) -> str:
-        if os.getenv("SENTRY_URL"):
-            return os.environ["SENTRY_URL"]
+    def SENTRY_BACKEND_DSN(self) -> str:
+        if os.getenv("SENTRY_BACKEND_DSN"):
```

Comment:
Where does this env var get set?

Reply:
It doesn't, it's left over from when I was trying to make Sentry play nice on a local setup. I don't expect this to ever work - so I will remove this check so no one gets confused!